diktat-rules/src/test/resources/test/paragraph3/long_line/LongLineExpressionTest.kt

```diff
@@ -0,0 +1,12 @@
+package test.paragraph3.long_line
```

Comment:
I suggest to decrease maximum length for tests to, say, 50 or 80 to make these examples more readable :)